packages/charts/src/components/MicroBarChart/MicroBarChart.tsx

```diff
@@ -61,6 +61,10 @@ export interface MicroBarChartProps
    * - `DataLabel`: a custom component to be used for the data label
    */
   measure: MeasureConfig;
+  /**
+   * An optional number for the maxValue of the valueBar
+  */
+  customMaxValue?: number;
```

Comment:
Please rename the prop to `maxValue` as this explains more that the user is in full control over the max value of the `MicroBarChart`.